File: webapp-net/Ish/Models/PublicationSiteMap.cs
using System.Collections.Generic;
using Sdl.Web.Common.Models;

namespace Sdl.Web.Modules.Ish.Models
{
    public class PublicationSiteMap : EntityModel
    {
        public int PublicationId { get; set; }

        public int NamespaceId { get; set; }

        public List<SiteMapUrlEntry> Urls { get; set; } = new List<SiteMapUrlEntry>();
    }
}

File: webapp-java/dxa-module-dynamic-documentation/src/main/java/com/sdl/dxa/modules/dd/models/PublicationSiteMap.java
package com.sdl.dxa.modules.dd.models;

import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;

import java.util.ArrayList;
import java.util.List;

public class PublicationSiteMap extends AbstractEntityModel {
    private int publicationId;
    private int namespaceId;
    private List<SiteMapUrlEntry> urls = new ArrayList<SiteMapUrlEntry>();

    public int getPublicationId() {
        return publicationId;
    }

    public void setPublicationId(int publicationId) {
        this.publicationId = publicationId;
    }

    public int getNamespaceId() {
        return namespaceId;
    }

    public void setNamespaceId(int namespaceId) {
        this.namespaceId = namespaceId;
    }

    public List<SiteMapUrlEntry> getUrls() {
        return urls;
    }

    public void setUrls(List<SiteMapUrlEntry> urls) {
        this.urls = urls;
    }
}
